lib/explorer/series.ex

```diff
@@ -2244,6 +2244,51 @@ defmodule Explorer.Series do
   @spec pow(left :: Series.t() | number(), right :: Series.t() | number()) :: Series.t()
   def pow(left, right), do: basic_numeric_operation(:pow, left, right)
 
+  @doc """
+  Calculate the exponent of a logarithm.
+
+  The resultant series is going to be of dtype :integer
+  if the series and the base are both integers. Otherwise
+  the it is going to be a float series.
```

Comment:
I guess the log function should always return float series. For Example $log_2 (3) \approx 1.58496$

Reply:
Agreed on always returning floats!